src/docs/examples/formioV2.html

```diff
@@ -0,0 +1,1480 @@
+            .formio_container { display: none !important;}
+          </style>
+        </noscript>
+
+        <!-- Not supported-->
+        <div class="alert alert-danger d-none" id="browser-warning" role="alert">
+          <p>Your browser is not supported. Some functionality might not work as expected.</p>
+          <p>Please upgrade your browser to a version on the
+            <a href="https://www.forgov.qld.gov.au/browser-support-lists">supported browser list</a></p>
+        </div>
+
+        <!-- Form/Loader-->
+        <div id="formio-279585" class="formio_container qg-forms-v2"
+             data-formio
+             data-formio-project-name="lwjbuevedlwgsem"
+             data-formio-form-name="submitcontenttopublishonforgovernment"
+             data-formio-form-confirmation="information-and-communication-technology/communication-and-publishing/website-and-digital-publishing/publish-on-queensland-government-websites/publish-on-for-government/submit-content-to-publish-on-for-government/thank-you-page"
+             data-formio-form-revision=""
+             data-formio-env-url="api.forms.platforms.qld.gov.au"
+             data-formio-pdf-download-message=""
+             data-formio-namespace=""
+             data-formio-createForm-options="formioCreateFormOptions"
+             data-formio-createForm-controller="formioCreateFormController"
+        >
+          <img src="https://oss-uat.clients.squiz.net/__data/assets/image/0019/126703/Spinner-1s-200px.png" />
+        </div>
+        <!--browser support script-->
+        <script type="text/javascript" src="/?a=147385"></script>
+        <script>
+          // choose which version of scripts to use, default to v1.x.x-latest if blank
+          window.formioQldCdnVersion = "";
+        </script>
+        <!-- PROD: load formio-script.js, comment this block if you want to test in UAT -->
+        <script src="https://static.qgov.net.au/formio-qld/v1/v1.x.x-latest/formio-script.min.js" type="text/javascript"></script>
+        <!-- end PROD -->
+        <!-- UAT: load formio-script.js from gitbridge far all forms/pages globally, uncomment this block if you want to test in UAT-->
+        <!--<script src="https://oss-uat.clients.squiz.net/__data/assets/git_bridge/0025/248740/formio-script.gitbridge.js?h=44e0f93" type="text/javascript"></script>-->
+        <!-- end UAT -->
+        <script>
+          FormioScript.init().then(() => {
+            // Formio global object initiated, you cound customise it here
+          });
+        </script>
+        <!-- #252693 formio-script.js: edit this file if you want to add custom options or controller to the form -->
+        <script src="/?a=252693" type="text/javascript"></script>
+        <script>
+          // Persistent fix for iPhone/Safari
+          window.onpageshow = function (event) {
+            if (event.persisted) {
+              window.location.reload();
+            }
+          };
+        </script>
+        <style>
```

Comment:
@asifamingov this style tag has been removed in squiz matrix component

Reply:
> @asifamingov this style tag has been removed in squiz matrix component

I deleted those examples of specific forms as those might cause some issues if those are not live and in the examples folder if someone finds those.